src/js/jsx/sections/libraries/assets/Color.jsx

```diff
@@ -36,25 +35,36 @@ define(function (require, exports, module) {
 
     var Color = React.createClass({
         mixins: [FluxMixin],
-
-        componentWillMount: function () {
-            var element = this.props.element,
-                representation = element.getPrimaryRepresentation(),
-                color = representation.getValue("color", "data"),
-                colorString = this._getStringColorValue(color),
-                hexValue = tinycolor(color.value).toHexString().toUpperCase();
-
-            this.setState({
-                colorData: color,
-                colorString: colorString,
-                hexValue: hexValue
-            });
+    
+        /**
+         * Find color asset's RGB representation and then return instance of Color model.
+         *
+         * @private
+         * @return {ColorModel}
+         */
+        _getColor: function () {
+            var representations = this.props.element.representations;
+            
+            for (var i = 0; i < representations.length; i++) {
```

Comment:
It would be better to use a library function for this instead of a `for` loop. Maybe something like [`_.find`](https://lodash.com/docs#find). Also, it's unclear to me, but it seems possible that this could return null/undefined. If so, the JSDoc should be amended appropriately.